hs_core/hydroshare/utils.py

```diff
@@ -7,7 +7,10 @@
 import logging
 import shutil
 
-from django.db.models import get_model, get_models
+from django.apps import apps
+get_model = apps.get_model
+from django.apps import apps
+get_models = apps.get_models
```

Comment:
why is this import done twice? Why are these statements in the import block? Neither of these are even used here, it seems.